src/amo/components/LandingPage.js

```diff
@@ -108,9 +109,29 @@ export class LandingPageBase extends React.Component {
     }
 
     const { addonType, html } = content;
+    const themeText = 'Change your browser’s appearance. Choose from thousands of themes to give Firefox the look you want.';
```

Comment:
These will need to be wrapped in `i18n.gettext()`. You can get `i18n` from props by adding it to line 96.